src/psyclone/psyir/nodes/node.py

```diff
@@ -1502,6 +1502,32 @@ def debug_string(self):
         from psyclone.psyir.backend.debug_writer import DebugWriter
         return DebugWriter()(self)
 
+    def path_from(self, ancestor):
+        ''' Find the path in the psyir tree between ancestor and node and
+        returns a list containing the path.
```

Comment:
Please give a bit more detail about how the resulting path can be used, e.g.:

    idx_list = node.path_from(ancestor)
    cursor = ancestor
    for idx in idx_list:
        cursor = cursor.children[idx]
    assert cursor is node

Reply:
Added